probe_scraper/scrapers/moz_central_scraper.py

```diff
@@ -133,7 +135,7 @@ def extract_tag_data(tag_data, channel, min_fx_version, max_fx_version):
     return results
 
 
-def download_files(channel, node, temp_dir, error_cache):
+def download_files(channel, node, temp_dir, error_cache, version):
```

Comment:
Looks like the tests are failing because of this change. Can you also add a test for it explicitly?

Reply:
Fixed the tests, and I think given one of the tests already tests nightly 71 it should be tested.